src/linear_algebra/affine_space.lean

```diff
@@ -195,10 +195,11 @@ structure affine_map (k : Type*) (V1 : Type*) (P1 : Type*) (V2 : Type*) (P2 : Ty
 namespace affine_map
 
 variables {k : Type*} {V1 : Type*} {P1 : Type*} {V2 : Type*} {P2 : Type*}
```

Comment:
Why don't we use unicode subscripts in this file?

Reply:
I can change this in another PR. I don't want to mix features with style.